### PR TITLE
Replace drill_hit.ogg and drill_use.ogg with better sounds

### PR DESCRIPTION
**Pull request by pissdemon**
## About the PR
The original drill_hit.ogg and drill_use.ogg (used by the power drill and medical drill, for instance) sound ear-piercingly shrill on some headphones because of loud very high frequency content at the end. They also aren't very good (what's that weird noise in drill_use.ogg?) and lack attribution info. This PR replaces them with better ones.

### Sound sources:

<ul>
     <li>S: <a href="https://freesound.org/people/MinecraftGamerLR/sounds/728696/">Bloody Stab 1</a> by <a href="https://freesound.org/people/MinecraftGamerLR/">MinecraftGamerLR</a> | License: <a href="https://creativecommons.org/licenses/by/4.0/">Attribution 4.0</li>
     <li>S: <a href="https://freesound.org/people/stomachache/sounds/262213/">Electric Drill</a> by <a href="https://freesound.org/people/stomachache/">stomachache</a> | License: <a href="http://creativecommons.org/publicdomain/zero/1.0/">Creative Commons 0</li>
</ul>

## Technical details
Just replaces two OGG files and adds attribution.

## Media
https://github.com/space-wizards/space-station-14/assets/165581243/c7964cdc-e166-40e5-ae07-af7353c47859

## Breaking changes
None.

**Changelog**
n/a, doesn't change gameplay (but saves your ears)

### CODE REVIEW

File: Resources/Audio/Items/attributions.yml [hunk 1]
```diff
@@ -107,3 +107,13 @@
   license: "CC0-1.0"
   copyright: "Bhijn and Myr (github username deathride58)"
   source: "https://github.com/space-wizards/space-station-14/pull/23476"
+
+- files: ["drill_hit.ogg"]
+  licence: "CC-BY-4.0"
```
Comment (mirrorcult) [inline Resources/Audio/Items/attributions.yml:112]:
```suggestion
  license: "CC-BY-4.0"
```

File: Resources/Audio/Items/attributions.yml [hunk 2]
```diff
@@ -107,3 +107,13 @@
...
+  copyright: "Created by vanilla with sounds from MinecraftGamerLR and stomachache on freesound.org"
+  source: "https://github.com/space-wizards/space-station-14/pull/26622"
+
+- files: ["drill_use.ogg"]
+  licence: "CC0-1.0"
```
Comment (mirrorcult) [inline Resources/Audio/Items/attributions.yml:117]:
```suggestion
  license: "CC0-1.0"
```

**Review by mirrorcult — DISMISSED** [review]
sounds seem fine to me

File: Resources/Audio/Items/attributions.yml (hunk 1 above)
Comment (metalgearsloth) [reply Resources/Audio/Items/attributions.yml:112]: england destroyed.

**Comment by metalgearsloth** [comment]
I'm indifferent.

**Pull request merged**